pages/en/anime/watch/[...info].js

```diff
@@ -474,6 +485,23 @@ export default function Watch({
           content={episodeNavigation?.playing?.description || info?.description}
         />
       </Head>
+      <RateModal
+        onSubmit={async (e) => {
+          e.preventDefault();
+          const rating = e.target.rating.value;
+          const review = e.target.review.value;
+          await markProgress({
+            mediaId: dataMedia.id,
+            progress: episodeNavigation.playing.number,
+            notes: review,
+            // scoreRaw is 1-100 while our input is 1-10
+            rating: 10 * rating,
+          });
```

Comment:
I think it would be better if you make a new function for rating instead of using markProgress

Reply:
What if i use markComplete?